pika/channel.py

```diff
@@ -390,7 +390,7 @@ def basic_reject(self, delivery_tag, requeue=True):
         """
         if not self.is_open:
             raise exceptions.ChannelClosed()
-        if not isinstance(delivery_tag, int):
+        if not is_integer(delivery_tag):
```

Comment:
I thought about using it, but decided against it because:

```py
>>> import numbers
>>> numbers.Integral.register(type(None))
>>> isinstance(None, numbers.Integral)
True
```